### ISSUE TITLE
"Media check failed" reported for specific videos

### ISSUE THREAD

**Issue by deldesir** [iiab/calibre-web#101]
**What we know so far**
- This happens to specific videos like https://www.youtube.com/watch?v=QXQrvT23rPw
- This causes metadata parsing
- The videos are downloaded without covers

Media check is done by xklb per:
https://github.com/chapmanjacobd/library/blob/7854404482cd8cb11ff435ac2d68e96a01d0e338/xklb/lb.py#L58

**Comment by holta** [comment]
@deldesir is this somewhat related to "YouTube Movies (ie. the YouTube_Movies_TV channel) are encrypted. Normal YouTube videos don't have this DRM" that Jacob helped us identify?

- https://github.com/iiab/calibre-web/issues/85#issuecomment-1876530655

**Comment by holta** [comment]
@deldesir can you summarize current status?

Should this ticket be closed and/or what issues remain here?

**Comment by deldesir** [comment]
This can be closed for now. This error happens when there's no available format for a submitted URL.

**Comment by holta** [comment]
@EMG70 can you please keep an eye on this?

i.e. please let us all know if such errors are not being communicated cleanly & clearly!
